string model_name_1
string link_name_1
---
bool ok
string model_name_2
string link_name_2